docs/rnn.rst

```diff
@@ -0,0 +1,211 @@
+Recurrent neural networks
+=========================
+
```

Comment:
I think it would be nice to put something like

```
This tutorial explains recurrent bricks in Blocks. If you are new to the brick concept, we recommend to start with https://blocks.readthedocs.org/en/latest/bricks_overview.html first and continue with this tutorial afterwards
```
(that was a bad and very quick wording and you can change it as you like).